src/main/java/com/example/auctionapp/model/Image.java

```diff
@@ -0,0 +1,43 @@
+package com.example.auctionapp.model;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+@Table(name = "image")
+public class Image extends Resource {
+
+    @NotBlank
+    private String url;
+
+    @ManyToOne
+    @JoinColumn(name = "product_id", referencedColumnName = "id", updatable = false)
+    private Product product;
```

Comment:
keep in mind that user profile will also contain images, maybe you should make this table more generic.

Reply:
Not tied to product